File: BugTrackerFrontend/src/Models/IssueModel.ts
import { IIssueModel } from "../types"
export default class IssueModel implements IIssueModel {
    public title!: string;
    public description!: string;
    public solved!: boolean;
    public status!: string;
    public label!: string;
    public priority!: string;

    constructor(params: IIssueModel) {
        Object.assign(this, params)
    }
}

File: BugTracker.DAL/Models/IssueModel.cs
namespace BugTracker.DAL.Models
{
    public class IssueModel
    {
        public string Title { get; set; }
        public string Description { get; set; }
        public bool Solved { get; set; }
        public string Status { get; set; }
        public string Priority { get; set; }
        public string Label { get; set; }

    }
}
